test/service/test_pagination.py

```diff
@@ -45,7 +45,7 @@ def assert_page1_correct(self, json_response):
         num_hits = len(json_response['hits'])
         if 'search_after' in json_response['pagination']:
             self.assertEqual(json_response['pagination']['search_after'],
-                             json_response['hits'][num_hits - 1]['entryId'],
+                             '"{}"'.format(json_response['hits'][num_hits - 1]['entryId']),
```

Comment:
We don't use format(), only f-strings or concatenation. The latter would look the the least weird in this case I think.

Reply:
Also, shouldn't this involve json.dumps/loads instead?

Reply:
I went with `format()` to match its use throughout that file, but yes `json.dumps()` is a better way to produce the verification values in this case.